Find404.scala

```diff
@@ -0,0 +1,61 @@
+import java.io._
+import scala.io.Source._
+import scala.io.StdIn._
+import scala.collection.mutable.ArrayBuffer
+
+var fileSet = ArrayBuffer[File]()
+var ignoreSet = ArrayBuffer[String]()
```

Comment:
All of these can be `val`s instead of `var`s. `val`s are immutable and generally preferred. Because you don't have to reason about if anyone who has access to the variable changed it.